initpost.sh

```diff
@@ -1,218 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Pls don't delete this

Reply:
Oh do we actually not want these deleted?